repotracker/wrappers.go

```diff
@@ -0,0 +1,100 @@
+package repotracker
+
+import (
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/model"
+	"github.com/evergreen-ci/evergreen/thirdparty"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/message"
+	"github.com/pkg/errors"
+)
+
+// use sentinel error values to provide compatibility with the original system
+var (
+	errProjectDisabled  = errors.New("project disabled")
+	errEncounteredError = errors.New("repotracker encountered error")
+)
+
+func CollectRevisionsForProject(conf *evergreen.Settings, project model.ProjectRef) error {
+	if !project.Enabled {
+		return errors.Wrap(errProjectDisabled, project.String)
```

Comment:
epotracker/wrappers.go:20:49: cannot use project.String (type func() string) as type string in argument to "github.com/evergreen-ci/evergreen/vendor/github.com/pkg/errors".Wrap